fault_injection/configs/pen.global_fi.crypto.kmac.cw310.yaml

```diff
@@ -35,10 +35,11 @@ fiproject:
   plot_y_axis: "glitch_width"
   plot_y_axis_legend: "[cycles]"
 test:
-  # which_test: "crypto_fi_kmac_key"
-  which_test: "crypto_fi_kmac_absorb"
-  # which_test: "crypto_fi_kmac_squeeze"
-  expected_result: '{"ciphertext":[184,34,91,108,231,47,251,27],"alerts":0}'
+  # which_test: "crypto_kmac_key"
+  which_test: "crypto_kmac_absorb"
+  # which_test: "crypto_kmac_squeeze"
+  # which_test: "crypto_kmac_static"
+  expected_result: '{"ciphertext":[184,34,91,108,231,47,251,27],"alerts":0,"err_status":0}'
```

Comment:
I changed the expected result in https://github.com/nasahlpa/opentitan/commit/53713111b147626c39ecc542ec17c36be48a82dc

`expected_result` is now: 

`{"digest":[184,34,91,108,231,47,251,27],"digest_2nd":[142,188,186,201,216,47,203,192],"alerts":[0,0,0],"err_status":0}`

Reply:
Updated, thanks!